.github/workflows/backport.yml

```diff
@@ -19,15 +19,13 @@ jobs:
         env:
           BACKPORT_LABEL_REGEXP: "backport/(?P<target>website)"
           BACKPORT_TARGET_TEMPLATE: "stable-{{.target}}"
-          BACKPORT_MERGE_COMMIT: true
```

Comment:
how about leaving these in but with `false` and a comment pointing to this issue

Reply:
Yeah, good idea. Added in 74ee31a8f